src/Workspaces/Core/Portable/Workspace/ApplyChangesKind.cs

```diff
@@ -24,6 +24,7 @@ public enum ApplyChangesKind
         ChangeDocumentInfo = 16,
         AddAnalyzerConfigDocument = 17,
         RemoveAnalyzerConfigDocument = 18,
-        ChangeAnalyzerConfigDocument = 19
+        ChangeAnalyzerConfigDocument = 19,
+        ChangeProjectInfo = 20
```

Comment:
i see reads of this... but i don't see anything returning this. is that expected?

Reply:
this is modelled the same way as `ChangeDocumentInfo`, basically it's never "returned", it's used as a flag for a `Workspace` implementation to decide which changes are supported and which aren't. I added a test that throws when the change is not supported

Reply:
Gotcha, given teh existence of ChangeDocInfo, and you followin the same pattern as it (i assume!) then this would be ok with me :)